Gemfile

```diff
@@ -34,6 +34,7 @@ group(:development, :test) do
   gem "ruby-shadow", :platforms => :ruby unless RUBY_PLATFORM.downcase.match(/(aix|cygwin)/)
 
   # For external tests
+gem "chef-zero", github: 'chef/chef-zero', branch: "jk/policies-cheffs"
```

Comment:
This is temporary (see last commit)--it's a branch of chef-zero without `--skip-policies` :)